README.md

```diff
@@ -3,12 +3,35 @@ Used to populate our backend (currently Firebase) for the Ember Help Wanted app
 
 
 ### Development
+- Set the environmental variables from the section below.
 - Use [ngrok](https://ngrok.com/) to create a tunnel to your machine that github can reach
 - In your repo, go to *Settings* -> *Webhooks & services* -> Add webhook
   - Payload URL: __YourNGrokUrl__/issue-handler
   - Secret: oursecrethere
 - To inspect payload visit http://127.0.0.1:4000
 - To inspect logs run ./node_modules/.bin/bunyan logs/events.log
 
+### Environment variables to setup
+```
+export OPENSHIFT_NODEJS_IP="127.0.0.1"
+export OPENSHIFT_NODEJS_PORT=8080
+export FIREBASE_APP="https://help-wanted-dev.firebaseio.com"
+export FIREBASE_SECRET="344SliTRyGVneG8QdRfCKRjF6wlkzFoHlpjNYHqY"
```

Comment:
Probably better to not have this exposed? You might want to reset the key too ...